Working on windows 10 docker integration

I tried to use it on windows 10 but if I do make build powershell gives some errors

`process_begin: CreateProcess(NULL, pwd, ...) failed.
Do not run Composer as root/super user! See https://getcomposer.org/root for details
Composer could not find a composer.json file in /app
To initialize a project, please create a composer.json file as described in the https://getcomposer.org/ "Getting Started" section
make: *** [composer-install] Erreur 1`

I fixed that just changing pwd and making it cross system compliant from $(shell pwd) to ${PWD}

I've removed $(id -u):$(id -g) too since such thing doesn't exist in docker for windows but we could create a volume with those arguments.

Or we could simply have two different make files one for windows and one for linux.

But even if I try to do all the changes to make it works I still getting some errors...
I changed 
`composer-install:
	@docker run --rm --interactive --tty --volume $(shell pwd):/app --user $(id -u):$(id -g) \
		gsingh1/prestissimo install \
			--ignore-platform-reqs \
			--no-ansi \
			--no-interaction`
to
`composer-install:
	@docker run --rm --interactive --tty --volume ${PWD}:/app gsingh1/prestissimo install --ignore-platform-reqs --no-ansi --no-interaction`

Still getting this error:
Do not run Composer as root/super user! See https://getcomposer.org/root for details
Composer could not find a composer.json file in /app
To initialize a project, please create a composer.json file as described in the https://getcomposer.org/ "Getting Started" section
make: *** [composer-install] Erreur 1

I don't know why because if I execute the command directly in powershell all seems ok:
`docker run --rm --interactive --tty --volume ${PWD}:/app gsingh1/prestissimo install --ignore-platform-reqs --no-ansi --no-interaction`

Maybe it's related to make but I don't really know much about it...

--- a/Makefile
+++ b/Makefile
@@ -1,16 +1,18 @@
 .PHONY: all deps composer-install composer-update build test run-tests
 
+current-dir := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+
 deps: composer-install
 
 composer-install:
-	@docker run --rm --interactive --tty --volume $(shell pwd):/app --user $(id -u):$(id -g) \
+	@docker run --rm --interactive --tty --volume $(current-dir):/app --user $(id -u):$(id -g) \
 		gsingh1/prestissimo install \
 			--ignore-platform-reqs \
 			--no-ansi \
 			--no-interaction
 
 composer-update:
-	@docker run --rm --interactive --tty --volume $(shell pwd):/app --user $(id -u):$(id -g) \
+	@docker run --rm --interactive --tty --volume $(current-dir):/app --user $(id -u):$(id -g) \
 		gsingh1/prestissimo update \
 			--ignore-platform-reqs \
 			--no-ansi \

